Mathlib/Tactic/CancelDenoms/Core.lean

```diff
@@ -154,63 +154,61 @@ def synthesizeUsingNormNum (type : Q(Prop)) : MetaM Q($type) := do
     throwError "Could not prove {type} using norm_num. {e.toMessageData}"
 
 /--
-`mkProdPrf α sα v tr e` produces a proof of `v*e = e'`, where numeric denominators have been
+`mkProdPrf α sα v v' tr e` produces a proof of `v'*e = e'`, where numeric denominators have been
 canceled in `e'`, distributing `v` proportionally according to the tree `tr` computed
 by `findCancelFactor`.
+
+The `v'` argument is a numeral expression corresponding to `v`, which we need in order to state
+the return ntype accurately.
```

Comment:
Should "ntype" be "type"? Or is "ntype" a concept I need to learn about?

Reply:
```suggestion
the return type accurately.
```

Reply:
The `n` is short for `tynpo`